simple-app-starter/monolithstarter-svc/src/main/java/com/validity/monolithstarter/service/DuplicateService.java

```diff
@@ -0,0 +1,366 @@
+package com.validity.monolithstarter.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.json.simple.JSONObject;
+import org.json.simple.JSONArray;
+
+@service
+public class DuplicateService {
+
+    /**
+     * Find the minimum of three values
+     *
+     * @param a - first value to compare
+     *
+     * @param b - second value to compare
+     *
+     * @param c - third value to compare
+     *
+     * @return the minimum value of the 3 parameters
+     */
+    public static int Min(int a, int b, int c){
+        int min = a; //Minimum distance, defaults to a
+
+        //Select minimum value from a, b, and c
+        if(b < min){
+            min = b;
+        }
+        if(c < min){
+            min = c;
+        }
+
+        return min;
+    }
+
+
+    /**
+     * Generic implementation of the Levenshtein algorithm to find the difference between two strings
+     * Based off of https://people.cs.pitt.edu/~kirk/cs1501/Pruhs/Spring2006/assignments/editdistance/Levenshtein%20Distance.htm
+     *
+     * @param s - first string to be compared
+     *
+     * @param t - second string to be compared
+     *
+     * @return the numeric differnce between the two strings
+     */
+    public static int Levenshtein(String s, String t){
```

Comment:
Created a service to handle detecting and reporting duplicates, as well as creating a json.